charts/customer-center/templates/service-backend.yaml

```
@@ -0,0 +1,17 @@
+apiVersion: v1
+kind: Service
+metadata:
+  name: {{ .Values.backend.service.name }}
```

I'm not sure this is a good idea. Why won't we just 
```suggestion
  name: {{ include "customerCenter.fullname" . }}-backend
```

Having multiple customer-center instances in the same namespace would not be possible with this "hardcoded to values" pattern.

---

I agree even if I think that will not happen. Let's update this.